niworkflows/viz/plots.py

```diff
@@ -24,33 +25,48 @@
 DINA4_LANDSCAPE = (11.69, 8.27)
 
 
-class fMRIPlot(object):
+class fMRIPlot:
     """
     Generates the fMRI Summary Plot
     """
-    __slots__ = ['func_file', 'mask_data',
-                 'tr', 'seg_data', 'confounds', 'spikes']
+    __slots__ = ('func_file', 'mask_data', 'tr', 'seg_data', 'confounds', 'spikes', '_cifti')
+
+    def __init__(
+        self,
+        func_file,
+        mask_file=None,
+        data=None,
+        conf_file=None,
+        seg_file=None,
+        tr=None,
+        usecols=None,
+        units=None,
+        vlines=None,
+        spikes_files=None,
+    ):
+        self._cifti = False
+        try:
+            func_img = nb.cifti2.load(func_file)
+            self._cifti = True
+        except (ImageFileError, HeaderDataError):
+            func_img = nb.load(func_file)
```

Comment:
Does `nb.load()` accept CIFTI2, @effigies ? If so, I guess it'd be more elegant to just load and determine whether this is cifti2 by looking at the instance's class.

Reply:
it does - sure.